src/v2/view-builder/views/SSOExtensionView.js

```diff
@@ -1,18 +1,21 @@
-import { loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import Util from '../../../util/Util';
 
+// for BETA,
+// redirect is needed for Apple SSO Extension to intercept the request, because
+// - XHR request is not interceptable
+// - form post is interceptable, but some app (Outlook) closes the app after
+// We may have a different approach/UX for EA
 const Body = BaseForm.extend({
   noButtonBar: true,
 
-  className: 'ion-form sso-extension-challenge',
-
-  title: loc('signin', 'login'),
+  title: 'You are being redirected',
```

Comment:
nit:
looks like it's recommendation from UX. would you mind to update `SuccessView.title` as well? maybe extra to login.properties.